Source/FullScreenPokemon.ts

```diff
@@ -10044,7 +10080,18 @@ module FullScreenPokemon {
         }
 
         /**
+         * Function to add an Item to the player's bag. If the Item
```

Comment:
Don't document this specifically for adding to the bag. It's a generic utility helper that *happens* to be used for the addItemToBag logic.